events/job_offmap.txt

```diff
@@ -1065,12 +1077,12 @@ character_event = {
 		if = {
 			limit = {
 				event_target:commander_improved = {
-					has_character_flag = heavy_infantry_leader
```

Comment:
Put heavy_infantry_leader to its original place so that the GitHub will stop thinking that you removed it.